dll/win32/uxtheme/png.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * PROJECT:     ReactOS uxtheme.dll
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     PNG image support in visual styles
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <Splitwirez@gmail.com>
+ */
+
+#include "uxthemep.h"
+
+#ifdef MSSTYLES_ACCEPT_PNG_IMAGES
+#include <ole2.h>
+#include <objidl.h>
+#include <gdiplus.h>
+#include <gdipluscolor.h>
+
+BOOL MSSTYLES_TryLoadPng(HINSTANCE hTheme, LPCWSTR szFile, LPCWSTR type, OUT HBITMAP *pngBitmap)
+{
+    HRSRC pngRes = FindResourceW(hTheme, szFile, type);
+    DWORD pngSize = SizeofResource(hTheme, pngRes);
+
+    HGLOBAL pngAlloc = GlobalAlloc(GMEM_MOVEABLE, pngSize);
+    if (!pngAlloc)
+        return FALSE;
+    
+    
+    LPCVOID pngData = LockResource(LoadResource(hTheme, pngRes));
+    LPVOID pngLock = GlobalLock(pngAlloc);
+    CopyMemory(pngLock, pngData, pngSize);
+    
+    IStream* pngStream = nullptr;
```

Comment:
Init not needed since it's going either to be initialized by the next call, or (in case it fails) you just return FALSE immediately.